src/groovy/org/pih/warehouse/api/StockMovement.groovy

```diff
@@ -250,7 +211,8 @@ class StockMovement {
                 dateCreated: shipment.dateCreated,
                 lastUpdated: shipment.lastUpdated,
                 requestedBy: shipment.createdBy,
-                shipment: shipment
+                shipment: shipment,
+                isFromOrder: shipment?.isFromPurchaseOrder
```

Comment:
Could we keep consistent naming in sm and in shipment for this field or will this require a big refactor?

Reply:
It's a small refactoring, but I did this on purpose. I left `isFromOrder` more generic on the stock movement in case there were other implementations, like production orders, transfer orders, etc. And I wanted to be more precise on the shipment side since these will always be from purchase orders. I'll commit it as-is for now but will think about refactoring it later.